scripts/access_control/bsc/access_control_eoa.py

```diff
@@ -0,0 +1,122 @@
+from brownie import accounts, ProxyAdminImpl, interface, Governable, Bank
+from brownie.exceptions import VirtualMachineError
+from ape_safe import ApeSafe
+
+ownable_contracts = [
+    "0x70df43522d3a7332310b233de763758adca14961",  # bank config
+    "0x8703f72dbdCD169A9C702E7044603ebbfb11425c",  # goblin config
+]
+goblins = [
+    "0x5a71a15037de71d52d8b27b6a637b617b4fae261",
+    "0xa25ad83285e0d0989b1409b3755be7e30bb821d7",
+    "0x44ae9dfaf7597c3665fb25ed31b5a32bab12ebc4",
+    "0x1601b34b8ae3378c7c2df675a37a7064164c74ca",
+    "0x22bb68bd0b113ccf688e0759ac0b4abc013df824",
+    "0xc07e4cc403b093bd94d80f30c78b63080a79cd10",
+    "0x8252aa78372b59d53e1c001d98f400611e9d2a20",
+    "0x7b4b9e3d52296416b7e0396593b5d885cdb27472",
+    "0x7e52d9dbaf0366aaec36175d551b21762336ecdb",
+    "0x78cffa390b2355b14aacaf97571da3be817c84bf",
+    "0xc2dc871a7ded817d8a960d2f2ceae6edb377c73f",
+    "0xaef73ea3a6f3f302af20a154c68996790bbaa85e",
+    "0x8f543b602e0bf1cc652c10f50d0ad853185a5f26",
+    "0xcd168c28c17aef8d47b88ff0904ef5bf1c806000",
+    "0xaa00f2b7dd0de46c6fc9655dbadd80ac91a66869",
+    "0x08d871ddad70bd3aef3fecfbf4350debc57d8264",
+    "0x549ef362657a3e3923793a494db3d89e3e5fda35",
+    "0x2f050b64ede3b1d21184435974bb1d2fe02012b6",
+    "0x3974071481dad49ac94ca1756f311c872ec3e26e",
+    "0xfdcdf8d07db8c5b33fbf46f41eced421d9d32bee",
+    "0x047683a9a7958c02ca86b6eecea1f8acfbd54f4f",
+    "0xc3c16508e77e99e67cfcd30b765e48a5a33d4c9d",
+    "0x3663aedebb70dcf0a64e2600233d6913dd3ecf2b",
+    "0xa0aa119e0324d864831c24b78e85927526e42d52",
+    "0x62e32e6ebeabf776b59f5dfb9b364779c3a64137",
+    "0x567f4a45d45945a75898be4cad299a8f32c86d08",
+]
+
+strats = [
```

Comment:
missing strats `0xe15e4a5c2b6ea78cc12e7d320b732924b64e6137` and `0xe47b5464a7860efd6486094f91c034a180108424`